src/inline_string.rs

```diff
@@ -674,6 +683,19 @@ mod tests {
 
         assert_eq!(s.insert(0, 'a'), Err(NotEnoughSpaceError));
     }
+
+    #[test]
+    fn test_write() {
+        use fmt::{Error, Write};
+
+        let mut s = InlineString::new();
+
+        for _ in 0..INLINE_STRING_CAPACITY {
+            assert!(write!(&mut s, "a").is_ok());
+        }
+
+        assert_eq!(write!(&mut s, "a"), Err(Error));
```

Comment:
And test that this doesn't modify its contents -- perhaps writing `"b"` here would make it easier.

Reply:
This is now handled by comparing the `s` with `normal_string`.
If the last `write!` were to modify the `s` in any way then the comparison would fail.

Reply:
Thinking back, yeah, writing "b" would've been better. I don't think we're in any danger though.  D